Source/Core/DolphinWX/Frame.cpp

```diff
@@ -690,6 +690,64 @@ WXLRESULT CFrame::MSWWindowProc(WXUINT nMsg, WXWPARAM wParam, WXLPARAM lParam)
 }
 #endif
 
+void CFrame::InhibitScreensaver()
+{
+// Inhibit the screensaver. Depending on the operating system this may also
+// disable low-power states and/or screen dimming.
+
+#if defined(HAVE_X11) && HAVE_X11
+  if (SConfig::GetInstance().bDisableScreenSaver)
```

Comment:
If this is an issue, I would prefer to leave this with someone else who wants this support since I have zero familiarity with systemd. (What about users without systemd / non-linux platforms?)